.github/workflows/mypy.yml

```diff
@@ -0,0 +1,41 @@
+name: Continuous Integration
```

Comment:
Use a better name

Reply:
@jxnl Renamed from `Continuous Integration` to `MyPy` for better readability.